### DISCUSSION TITLE
Way to change calibration?

### DISCUSSION THREAD

**Discussion by ChrisyboiG** [KeyPuncher/WiinUPro#164] [Help] [answered]
Is there a way to change the Wii remote calibration. Currently it only lets me have tilt controls with the remote straight up, but I want to use it on its side in a Wii Wheel. Can anyone help?

**Comment by KeyPuncher** [comment, marked as answer]
The orientation of the Wii remote doesn't matter, the accelerometer angles are absolute. For the purposes of using the Wheel it will either be the pitch or roll you'll you want.
